packages/cli/build.gradle.kts

```diff
@@ -1363,6 +1372,7 @@ val jvmDefs = mutableMapOf(
   "elide.kotlin.version" to libs.versions.kotlin.sdk.get(),
   "elide.kotlin.verbose" to "false",
   "elide.nativeTransport.v2" to enableNativeTransportV2.toString(),
+  "elide.gvmResources" to gvmResourcesPath,
```

Comment:
preferring this so paths never end up hard-coded; at build time, this property is used. at runtime, the resources path is calculated from the binary's own path.